### DISCUSSION TITLE
Double colon notation from multiline code blocks treated as existing column

### DISCUSSION THREAD

**Discussion by 23tux** [RafaelGB/obsidian-db-folder#415] [Q&A]
### Contact Details

hubert@hoelzl.io

### What happened?

I have a lot of multiline Ruby code in my vault like this:

```ruby
# uncomment this lines to get a proper backtrace from GraphQL::Backtrace
use GraphQL::Execution::Interpreter
```

When I'm adding a new column, the dropdown "Select an existing column to add" shows `uncomment this lines to get a proper backtrace from GraphQL` or `use GraphQL` as an existing column. Overall, this gets pretty noisy:
![Screenshot 2022-09-21 at 13 17 40](https://user-images.githubusercontent.com/38003/191490662-3480b320-440a-406d-9a50-f17f2eff86f9.png)

Is there a way to fix this?

### What platform were you using?

Desktop

### Version of the plugin

2.6.4

### Are you using the latest version of the plugin in your Obsidian vault?

- [X] I have verified that I am on the latest version

### Are you check if there is a similar issue?

- [X] I have verified that there are not similar issues

**Comment by RafaelGB** [comment]
This problem should be discussed in the Dataview forum. I don't know if there is a way to prevent it from collecting information if it is inside code blocks.

This plugin is a dataview search engine frontend

**Comment by RafaelGB** [comment]
FYI, We want to add a search filter of that existing column selector soon. Maybe if dv can not filter those fields, that option could be helpful
